### PR TITLE
Added more known buttons for webostv

### PR DESCRIPTION
**Pull request by Xinne**
Cant test them all on my device, but it might help :-)

## Proposed change
Added more known buttons for the webostv integration 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

none

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @Xinne, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#21601).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/webostv.markdown [hunk 1]
```diff
@@ -119,7 +119,40 @@ Available services: `button`, `command`
...
 | ---------------------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `entity_id`            | no       | Target a specific webostv media player.                                                                                                                                                                                                                                                |
-| `button`               | no       | Name of the button. Known possible values are `LEFT`, `RIGHT`, `DOWN`, `UP`, `HOME`, `MENU`, `BACK`, `ENTER`, `DASH`, `INFO`, `ASTERISK`, `CC`, `EXIT`, `MUTE`, `RED`, `GREEN`, `BLUE`, `VOLUMEUP`, `VOLUMEDOWN`, `CHANNELUP`, `CHANNELDOWN`, `PLAY`, `PAUSE`, `NETFLIX`, `GUIDE`, `AMAZON`, `0`, `1`, `2`, `3`, `4`, `5`, `6`, `7`, `8`, `9` |
+| `button`               | no       | Name of the button. Known possible values are listed below. |
+
+
+#### Known button values ####
+
+| Button type | Buttons | Description |
```
Comment (thecode) [inline source/_integrations/webostv.markdown:127]:
```suggestion
| Button type | Values | Description |
```

File: source/_integrations/webostv.markdown [hunk 2]
```diff
@@ -119,7 +119,40 @@ Available services: `button`, `command`
...
+| ---------------------- | -------- | -------------------------------------------------------------------------------------------------- |
+| Home | HOME | | 
+| Numbers            | 1, 2, 3, 4, 5, 6, 7, 8, 9, 0       | Keypad numbers     |
+| Volume              | MUTE, VOLUMEUP, VOLUMEDOWN       |   |
```
Comment (thecode) [inline source/_integrations/webostv.markdown:131]: Please wrap each option with `
```suggestion
| Volume              | `MUTE`, `VOLUMEUP`, `VOLUMEDOWN`       |   |
```

File: source/_integrations/webostv.markdown [hunk 3]
```diff
@@ -119,7 +119,40 @@ Available services: `button`, `command`
...
+| Aspect ratio | ASPECT_RATIO | |
+| Screen remote | SCREEN_REMOTE | This is the three-dots button on magic remotes |
+| Teletext | TELETEXT | |
+| BS | BS, BS_NUM_# (1-12) | eg BS_NUM_1| 
+| CS1 | CS1, CS1_NUM_# (1-12) | eg CS1_NUM_1| 
+| CS2 | CS2, CS2_NUM_# (1-12) | eg CS2_NUM_1| 
+| TER | TER, CS2_NUM_# (1-12) | eg TER_NUM_1| 
+| 3D | 3D_MODE| |
+| Apps | NETFLIX, AMAZON, ALEXA ||
+| Unknown/untested| 3DIGIT_INPUT, GOTOPREV, GOTONEXT, SAP, BML_DATA, JAPAN_DISPLAY,TEXTOPTION, ASTERISK | |
```
Comment (thecode) [inline source/_integrations/webostv.markdown:153]: We should not document untested or unknown options

File: source/_integrations/webostv.markdown [hunk 4]
```diff
@@ -119,7 +119,40 @@ Available services: `button`, `command`
...
+| Color buttons | RED, GREEN, YELLOW, BLUE | |
+| Menu/apps | MENU, QMENU, FAVORITES, MYAPPS, RECENT | |
+| Zoom | LIVE_ZOOM, MAGNIFIER_ZOOM | |
+| Aspect ratio | ASPECT_RATIO | |
+| Screen remote | SCREEN_REMOTE | This is the three-dots button on magic remotes |
+| Teletext | TELETEXT | |
+| BS | BS, BS_NUM_# (1-12) | eg BS_NUM_1| 
+| CS1 | CS1, CS1_NUM_# (1-12) | eg CS1_NUM_1| 
+| CS2 | CS2, CS2_NUM_# (1-12) | eg CS2_NUM_1| 
+| TER | TER, CS2_NUM_# (1-12) | eg TER_NUM_1|
```
Comment (thecode) [inline source/_integrations/webostv.markdown:150]: I don't understand what are these options

**Review by thecode — COMMENTED** [review]
Can you point me to a reference for the list of this options, I don't want to document unsupported options as they end up in issues

**Comment by frenck** [comment]
Since there was no response at all, I'm declaring this PR stale and closing it.
Feel free to re-open this PR when you feel like that is the appropriate thing to do.

**Pull request closed without merging**